cache/src/test/scala/CacheTest.scala

```diff
@@ -12,7 +12,7 @@ object CacheTest // extends Properties("Cache test")
   import FileInfo.hash._
   import Ordering._
   import sbinary.DefaultProtocol.FileFormat
-  def test {
+  def test() {
```

Comment:
As you know, Scala doesn't like function returning `Unit` with no params and parens.